File: Python Code/Daily_Flash_Code_0/Week 2/25Jan/11-DailyFlash_Solutions/11-DailyFlash_Solutions/25_Jan_Solutions_One/Java/Pattern.java
/*
Program 4: Write a Program to Print following Pattern.
a
A A 
a a a 
A A A A
a a a a a
*/


class Pattern{

	
	public static void main(String args[]){

		
		for(int outerItr = 0 ; outerItr < 5 ; outerItr++){

			for(int innerItr = 0 ; innerItr <= outerItr ; innerItr++){

				System.out.print(outerItr%2==0?"a ":"A ");
			}
			System.out.printf("\n");


		}	
	}	
}	

File: Python Code/Daily_Flash_Code_0/Week 2/25Jan/11-DailyFlash_Solutions/11-DailyFlash_Solutions/25_Jan_Solutions_One/Python/pattern.py

"""
/*
Program 4: Write a Program to Print following Pattern.
a
A A 
a a a 
A A A A
a a a a a
*/

"""


for itr in range(5):
    for inner in range(itr+1):
        print("a " if itr%2==0 else "A ",end=" ")
    print("")    
